pos_sequence_ref_number/static/src/js/main.js

```diff
@@ -0,0 +1,80 @@
+odoo.define('pos_sequence_ref_number.pos_sequence_ref_number_assets_backend', function(require) {
+    "use strict";
+
+    var models = require('point_of_sale.models');
+
+    var PosModelSuper = models.PosModel;
+    var posmodel_super = models.PosModel.prototype;
+    var _super_order = models.Order.prototype;
+
+
+    var sequence_next = function(seq){
```

Comment:
This method could be a part of the PosModel? Because there is no way to inherit this method at the moment.

Reply:
@benwillig This is a good idea. I will see how to do this.